x-pack/test_serverless/api_integration/test_suites/common/security/authorization.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const svlCommonApi = getService('svlCommonApi');
+  const supertest = getService('supertest');
+
+  describe('security/authorization', function () {
+    describe('route access', () => {
+      describe('disabled', () => {
+        it('get all priviledges', async () => {
+          const { body, status } = await supertest
+            .get('/api/security/privileges')
+            .set(svlCommonApi.getInternalRequestHeader());
+          svlCommonApi.assertApiNotFound(body, status);
+        });
+
+        it('get built-in elasticsearch priviledges', async () => {
```

Comment:
__typo:__ (here and below)
```suggestion
        it('get built-in elasticsearch privileges', async () => {
```